app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_post_comments_count
+
+  private
+
+  def update_post_comments_count
+    post_comments_count = Comment.where(post_id:).count
+    post.update(commentsCounter: post_comments_count)
+  end
```

Comment:
:orange_circle: [OPTIONAL] Below is a more concise and clear way to achieve the same result as the previous code:
```

def update_post_comments_count
    post.increment!(:commentsCounter)
end
```